audio-to-text/client/Containerfile

```diff
@@ -1,8 +1,8 @@
-FROM registry.access.redhat.com/ubi9/python-39:latest
+FROM registry.access.redhat.com/ubi9/python-311:latest
```

Comment:
Is it possible to pin the version? Using the latest tag might cause some issues

Reply:
Same comment is applied to everywhere the latest tag is being used. Renovate can be onboarded and used for bumping the versions.

Reply:
I updated to pin all python versions